webui/react/src/components/ExperimentActionDropdown.tsx

```diff
@@ -189,7 +212,8 @@ const ExperimentActionDropdown: React.FC<Props> = ({
     <div className={css.base} title="Open actions menu" onClick={stopPropagation}>
       <Dropdown overlay={menu} placement="bottomRight" trigger={[ 'click' ]}>
         <button onClick={stopPropagation}>
-          <Icon name="overflow-vertical" />
+          {settings.pinned.includes(id) ?
+            <PushpinOutlined /> : <Icon name="overflow-vertical" />}
```

Comment:
nit: Feels kinda weird to me that you click on the pin to bring up the overflow menu? If this went through alex then don't worry about it, just thought I'd mention

Reply:
Its just my idea. I wanted to change icon when its hovered but I forgot to do that. But let me ask him first

Reply:
I talked to alex and added border in the last item